cmd/bee/cmd/start.go

```diff
@@ -321,6 +321,7 @@ func (c *command) configureSigner(cmd *cobra.Command, logger logging.Logger) (co
 
 		logger.Infof("using swarm network address through clef: %s", address)
 	} else {
+		logger.Warning("clef is not enabled; portability and security of your keys is sub optimal")
```

Comment:
WDYM with `portability`? consider `clef is not enabled; we encourage using clef to ensure the safety of your keys`

Reply:
Thank you for the suggestion! With portability, I mean that using your swarm keys in any wallet that is not Swarm is difficult to achieve. This is solved when the user uses clef.